administrator/components/com_admin/sql/updates/mysql/6.0.0-2025-03-27.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `ordering`, `state`) VALUES
+(0, 'plg_fields_note', 'plugin', 'note', 'fields', 0, 1, 1, 0, 1, '', '{"class":"alert alert-info","heading":"h4"}', '', 17, 0);
```

Comment:
```suggestion
(0, 'plg_fields_note', 'plugin', 'note', 'fields', 0, 1, 1, 0, 1, '', '{"class":"alert alert-info","heading":"h4"}', '', -1, 0);
```
New plugins should be installed on update with order number -1 so they are shown at the top.

Reply:
And the script has to be renamed, see my other comment in the main thread.

Reply:
ok i did so. thanks :D